README.md

```diff
@@ -1 +1,59 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+In this project, it includes building a simple HTML LeaderBoard tasks using webpack and served by a webpack dev server.
+
+# Built with
+
+<ul>
+<li>HTML</li>
+<li>CSS</li>
+<li>JavaScript</li>
+  <li>Webpack</li>
+</ul>
+
+# Prerequisites
+
+<ul>
+<li>VSCode</li>
+<li>Git</li>
+<li>Browser With Developer Tools</li>
+</ul>
+
+# Project Setup
+
+1. Setup the project by creating a new directory and initializing it with npm using the official webpack documentation to set up webpack.
+2. Add the HTML to index.html in src folder.
+3. Worked on styling the website using style.css.
+4. Created modules for the project and imported them to index.js
+5. fix linters errors.
+6. test the website by adding and removing tasks.
+
+## Live Demo
+
+[Live Demo Link]
+
+# Run Tests
+
+I tested the website by running the code to a live server and opened it with Mozilla Firefox. Tested the local server.
+
+# Authors :bookmark_tabs:
+
+Elie Harfouche
+
+<ul>
+<li>Github <a href="https://github.com/X-Elie-X">Link To Github Profile</a></li>
+
+</ul>
+
+# Contributing :handshake:
+
+Contributions, issues, and feature request are welcome anytime.
+
+# Show your support
+
+Give a Like or a Star if you like this project.
+
+# Acknowledgements
+
+<ul>
+<li>Microverse</li>
```

Comment:
- [ ] Please update your Readme, it will be nice to add this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. Check out this [template](https://github.com/microverseinc/readme-template) for more clarifications